File: C++/Funsum.cpp
#include<iostream>
using namespace std;
int sum(int a,int b)
{
    cout<<"The sum of the two variables using function is:"<<a+b;
}
int main()
{
    int a,b;
    cout<<"Enter the two numbers:";
    cin>>a>>b;
    sum(a,b);
    return 0;
}

File: Java/Funsum.java
import java.util.Scanner;

public class Funsum {
    static void sum(int a,int b){
        System.out.println("The sum of the given two numbers is:"+(a+b));
    }
    public static void main(String arg[]){
        int a,b;
       try (Scanner s=new Scanner(System.in)){
        System.out.println("Enter the two numbers:");
        a=s.nextInt();
        b=s.nextInt();
       }
        sum(a,b);
    }
}
